doc/interop-test-descriptions.md

```diff
@@ -1011,16 +1011,49 @@ languages. Therefore they are not part of our interop matrix.
 #### rpc_soak
```

Comment:
This is in an "Experimental Tests" section. If we are actually talking about implementing this more widely, it seems we should move it out of the experimental section.

Reply:
moved out of experimental section